WordPress/src/main/java/org/wordpress/android/ui/prefs/DeleteSiteDialogFragment.java

```diff
@@ -108,14 +108,13 @@ private Spannable confirmationPromptString() {
     private void configureUrlConfirmation(AlertDialog.Builder builder) {
         View view = getActivity().getLayoutInflater().inflate(R.layout.delete_site_dialog, null);
         mUrlConfirmation = (EditText) view.findViewById(R.id.url_confirmation);
-        mUrlConfirmation.setHint(mSiteDomain);
```

Comment:
Haha, I was going to type "Because you didn't like it" until I realized I read this comment wrong: 
```
"The "Primary domain" label here seems unnecessary. If we decide to keep it, the spacing needs to be tweaked a bit." incorrectly.
```

I'll bring it back